[KNI] fix repo's url in verify-commits.sh script

Tal-or opened the pull request:
since we transfer the repo under the openshift-kni org, we need to set the url accordingly

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

fromanirh commented:
/hold
WIP: it compiles fine, but didn't run the tests yet.

fromanirh commented on pkg/noderesourcetopology/pluginhelpers.go:
```diff
@@ -19,9 +19,11 @@ package noderesourcetopology
+	"os"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
:\

Tal-or commented:
AFAIK @swatisehgal had issues with the ```InClusterConfig()``` we need to have E2E test to verifies those changes

fromanirh commented:
> AFAIK @swatisehgal had issues with the `InClusterConfig()` we need to have E2E test to verifies those changes

yup. I'll lift the WIP once I have a integration/e2e test. Manual testing was positive so far.

fromanirh commented:
both unit and integration test pass locally, but I think we need a e2e test.

fromanirh commented:
on second thought, this should probably sent upstream.

swatisehgal commented:
Yes, I agree but with preliminary tests on my cluster this is not working as expected. I am deep diving into this to figure out the root cause.

swatisehgal commented:
@fromanirh You mentioned above that the unit tests are running fine for you. However, I see the following when I run the unit tests:

```
$ make unit-test
hack/update-vendor.sh
warning: ignoring symlink /root/go/src/sigs.k8s.io/scheduler-plugins/manifests/install/charts/as-a-second-scheduler/templates/crds
warning: ignoring symlink /root/go/src/sigs.k8s.io/scheduler-plugins/manifests/install/charts/as-a-second-scheduler/templates/crds
hack/update-generated-openapi.sh
Generating Kubernetes OpenAPI
hack/unit-test.sh
?   	sigs.k8s.io/scheduler-plugins/cmd/controller	[no test files]
?   	sigs.k8s.io/scheduler-plugins/cmd/controller/app	[no test files]
I1109 14:14:18.396577 1525310 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:20.044069 1525310 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:20.044114 1525310 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:20.044139 1525310 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
I1109 14:14:21.610554 1525310 serving.go:347] Generated self-signed cert in-memory
--- FAIL: TestSetup (6.66s)
    --- FAIL: TestSetup/single_profile_config_-_NodeResourceTopologyMatch_with_args (1.56s)
        main_test.go:242: failed to create listener: failed to listen on 0.0.0.0:10251: listen tcp 0.0.0.0:10251: bind: address already in use
FAIL
FAIL	sigs.k8s.io/scheduler-plugins/cmd/noderesourcetopology-plugin	6.809s
I1109 14:14:19.772558 1525940 serving.go:347] Generated self-signed cert in-memory
I1109 14:14:20.818954 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:22.422821 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:22.422911 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:22.422954 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
I1109 14:14:24.002652 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:25.278711 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:25.311469 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:25.311574 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
I1109 14:14:27.680572 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:29.136266 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:29.136300 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:29.136321 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
W1109 14:14:29.154139 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:29.162788 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:30.314868 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:30.708594 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
I1109 14:14:31.532808 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:32.561034 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:33.015704 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:34.993547 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:34.993585 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:34.993620 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
W1109 14:14:35.001036 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:35.001036 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:36.057285 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:36.177903 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:36.725942 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:38.127960 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:38.128590 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:38.821504 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
I1109 14:14:41.151851 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:42.294287 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:43.412821 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:43.413113 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:43.413144 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
W1109 14:14:44.090323 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
I1109 14:14:45.014000 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:45.437145 1525940 reflector.go:436] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: watch of *v1alpha1.PodGroup ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
W1109 14:14:46.126711 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:46.126801 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:46.126843 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
W1109 14:14:46.545057 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
I1109 14:14:47.063810 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:48.329110 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:48.329142 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:48.329163 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
E1109 14:14:48.335549 1525940 targetloadpacking.go:120] "Unable to populate metrics initially" err="Get \"http://deadbeef:2020/watcher\": dial tcp: lookup deadbeef on 192.168.0.1:53: no such host"
I1109 14:14:49.212909 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:50.343363 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:50.343410 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:50.343432 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
E1109 14:14:50.349929 1525940 collector.go:140] "Load watcher client failed" err="Get \"http://deadbeef:2020/watcher\": dial tcp: lookup deadbeef on 192.168.0.1:53: no such host"
E1109 14:14:50.349972 1525940 collector.go:84] "Unable to populate metrics initially" err="Get \"http://deadbeef:2020/watcher\": dial tcp: lookup deadbeef on 192.168.0.1:53: no such host"
W1109 14:14:50.672843 1525940 reflector.go:436] k8s.io/client-go/informers/factory.go:134: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
I1109 14:14:51.776147 1525940 serving.go:347] Generated self-signed cert in-memory
W1109 14:14:52.689348 1525940 authentication.go:308] No authentication-kubeconfig provided in order to lookup client-ca-file in configmap/extension-apiserver-authentication in kube-system, so client certificate authentication won't work.
W1109 14:14:52.689425 1525940 authentication.go:332] No authentication-kubeconfig provided in order to lookup requestheader-client-ca-file in configmap/extension-apiserver-authentication in kube-system, so request-header client certificate authentication won't work.
W1109 14:14:52.689453 1525940 authorization.go:184] No authorization-kubeconfig provided, so SubjectAccessReview of authorization tokens won't work.
I1109 14:14:52.692268 1525940 plugin.go:90] "Creating new TopologyMatch plugin"
I1109 14:14:52.692306 1525940 pluginhelpers.go:45] "Loading kube client config from path" path="/tmp/scheduler-options289064928/config.kubeconfig" masterOverride=""
W1109 14:14:52.695872 1525940 reflector.go:436] github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/informers/externalversions/factory.go:119: watch of *v1alpha1.NodeResourceTopology ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
I1109 14:14:52.797007 1525940 plugin.go:129] "EventsToRegister called"
--- FAIL: TestSetup (34.54s)
    --- FAIL: TestSetup/default_config (1.52s)
        main_test.go:665: failed to create listener: failed to listen on 0.0.0.0:10251: listen tcp 0.0.0.0:10251: bind: address already in use
FAIL
FAIL	sigs.k8s.io/scheduler-plugins/cmd/scheduler	34.736s
?   	sigs.k8s.io/scheduler-plugins/pkg/apis/config	[no test files]
ok  	sigs.k8s.io/scheduler-plugins/pkg/apis/config/scheme	0.113s
ok  	sigs.k8s.io/scheduler-plugins/pkg/apis/config/v1beta1	0.033s
?   	sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/scheme	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1	[no test files]
ok  	sigs.k8s.io/scheduler-plugins/pkg/capacityscheduling	0.221s
ok  	sigs.k8s.io/scheduler-plugins/pkg/controller	3.293s
ok  	sigs.k8s.io/scheduler-plugins/pkg/coscheduling	0.202s
ok  	sigs.k8s.io/scheduler-plugins/pkg/coscheduling/core	1.095s
ok  	sigs.k8s.io/scheduler-plugins/pkg/crossnodepreemption	0.242s
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned/fake	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned/scheme	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned/typed/scheduling/v1alpha1	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned/typed/scheduling/v1alpha1/fake	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/internalinterfaces	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/scheduling	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/scheduling/v1alpha1	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/generated/listers/scheduling/v1alpha1	[no test files]
ok  	sigs.k8s.io/scheduler-plugins/pkg/noderesources	0.040s
ok  	sigs.k8s.io/scheduler-plugins/pkg/noderesourcetopology	0.031s
ok  	sigs.k8s.io/scheduler-plugins/pkg/podstate	0.133s
ok  	sigs.k8s.io/scheduler-plugins/pkg/qos	0.033s
ok  	sigs.k8s.io/scheduler-plugins/pkg/trimaran	0.059s
ok  	sigs.k8s.io/scheduler-plugins/pkg/trimaran/loadvariationriskbalancing	0.149s
ok  	sigs.k8s.io/scheduler-plugins/pkg/trimaran/targetloadpacking	0.171s
ok  	sigs.k8s.io/scheduler-plugins/pkg/util	0.017s
FAIL
make: *** [Makefile:123: unit-test] Error 1
```

What is particularly standing out to me in the above is the following:
```
W1109 14:14:52.695872 1525940 reflector.go:436] github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/informers/externalversions/factory.go:119: watch of *v1alpha1.NodeResourceTopology ended with: an error on the server ("unable to decode an event from the watch stream: got invalid watch event type: ") has prevented the request from succeeding
```
Looks like NRT objects are not being watched by the scheduler plugin here.

fromanirh commented:
better to consume the u/s facilities introduced in 1.22.

Tal-or committed:
[KNI] fix repo's url in verify-commits.sh script

since we transfer the repo under the openshift-kni org, we need to set the url accordingly

Signed-off-by: Talor Itzhak <titzhak@redhat.com>
